.github/workflows/lte-integ-test-bazel-magma-deb.yml

```diff
@@ -20,6 +20,11 @@ on:
   repository_dispatch:
     types: [magma-debian-artifact]
 
+env:
+  BAZEL_CACHE: bazel-cache
```

Comment:
Perhaps it would be easier to put the `.` inside of a string in the env already, e.g.
```
BAZEL_CACHE_NAME: 'bazel-cache'
BAZEL_CACHE_FOLDER: '.bazel-cache'
```
and then use the name for the key and the folder for everything else. Nesting is not possible here I think. But I would also be okay hard-coding the key below. Same for the repo cache. What do you think?

Reply:
refactored into action and handled there